execution.bs

```diff
@@ -8634,47 +8570,41 @@ request but has no recommendataions at present.</span>
               the `ref_count` to `0` then synchronizes with
               the call to <code><i>dec-ref</i>()</code> that decrements `ref_count` to `0`.
 
-8. For each type `split_t` and `ensure_started_t`, there is a different,
-    associated exposition-only implementation tag type, <i>`split-impl-tag`</i>
-    and <i>`ensure-started-impl-tag`</i>, respectively. Let
-    <i>`shared-impl-tag`</i> be the associated implementation tag type of
-    <i>`shared-cpo`</i>. Given an expression `sndr`, the expression
-    <code><i>shared-cpo</i>.transform_sender(sndr)</code> is equivalent to:
```

Comment:
this and the below is wrong. we need a _`split-impl-tag`_, and the _`impls-for`_ below needs to be specialized for it.